extensions/binding/org.eclipse.smarthome.binding.wemo/ESH-INF/thing/thing-types.xml

```diff
@@ -324,7 +324,7 @@
      <channel-type id="brewing" advanced="true">
         <item-type>DateTime</item-type>
         <label>Brewing</label>
-        <description>tba</description>
+        <description>This channel is not documented right now. If You find a valid description, please post to community</description>
```

Comment:
Err. I would suggest to REMOVE the channel from the binding as long as nobody knows what it is good for...